.ci/magic-modules/generate-terraform.sh

```diff
@@ -58,11 +58,7 @@ fi
 
 pushd "build/$SHORT_NAME"
 
-# go mod vendor is a very expensive operation.
-# If no changes, avoid running.
-if git diff-index --quiet HEAD --; then
```

Comment:
Can we change to `git diff-index --quiet master --` instead?

Reply:
My read here is that by using `HEAD` we were always gonna have no diff, because we committed right before. But by using `master` we'll see a diff if we've created a non-empty commit?